unable to build 0.21.3 for Windows - temporary value dropped while borrowed

receiving these errors when trying to `cargo build --release` from git branch 0.21.3:

```bash
error[E0716]: temporary value dropped while borrowed                                                                                                               
  --> src\subcommand\verify.rs:35:8
   |
34 |       } else if let Some(file) = &self.file {
   |  ____________-
35 | |       &fs::read(file)?
   | |        ^^^^^^^^^^^^^^^ creates a temporary value which is freed while still in use
36 | |     } else {
   | |     - temporary value is freed at the end of this statement
37 | |       unreachable!()
38 | |     };
   | |_____- borrow later used here
   |
   = note: consider using a `let` binding to create a longer lived value

error[E0716]: temporary value dropped while borrowed
  --> src\subcommand\wallet\sign.rs:38:8
   |
37 |       } else if let Some(file) = &self.file {
   |  ____________-
38 | |       &fs::read(file)?
   | |        ^^^^^^^^^^^^^^^ creates a temporary value which is freed while still in use
39 | |     } else {
   | |     - temporary value is freed at the end of this statement
40 | |       unreachable!()
41 | |     };
   | |_____- borrow later used here
   |
   = note: consider using a `let` binding to create a longer lived value

For more information about this error, try `rustc --explain E0716`.
error: could not compile `ord` (lib) due to 2 previous errors
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -8,7 +8,7 @@ autotests = false
 homepage = "https://github.com/ordinals/ord"
 repository = "https://github.com/ordinals/ord"
 autobins = false
-rust-version = "1.80.0"
+rust-version = "1.79.0"
 
 [package.metadata.deb]
 copyright = "The Ord Maintainers"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -8,7 +8,7 @@ autotests = false
 homepage = "https://github.com/ordinals/ord"
 repository = "https://github.com/ordinals/ord"
 autobins = false
-rust-version = "1.76.0"
+rust-version = "1.80.0"
 
 [package.metadata.deb]
 copyright = "The Ord Maintainers"

--- a/README.md
+++ b/README.md
@@ -129,7 +129,7 @@ cargo build --release
 
 Once built, the `ord` binary can be found at `./target/release/ord`.
 
-`ord` requires `rustc` version 1.80.0 or later. Run `rustc --version` to ensure
+`ord` requires `rustc` version 1.79.0 or later. Run `rustc --version` to ensure
 you have this version. Run `rustup update` to get the latest stable release.
 
 ### Docker

--- a/README.md
+++ b/README.md
@@ -129,7 +129,7 @@ cargo build --release
 
 Once built, the `ord` binary can be found at `./target/release/ord`.
 
-`ord` requires `rustc` version 1.76.0 or later. Run `rustc --version` to ensure
+`ord` requires `rustc` version 1.80.0 or later. Run `rustc --version` to ensure
 you have this version. Run `rustup update` to get the latest stable release.
 
 ### Docker

